src/metabase/email/_header.mustache

```diff
@@ -1,21 +1,53 @@
 <html>
 <head>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,900;1,100;1,300;1,400;1,700;1,900" rel="stylesheet">
   <style>
+    * {
+      box-sizing: border-box;
+    }
```

Comment:
https://www.caniemail.com/search/?s=box-sizing

Unfortunately, it's not supported in Outlook on Windows, Yahoo or AOL. 😩 

I think we should definitely strive to support Outlook at minimum. Granted, I don't think our current template or static viz code renders nicely on Outlook, but this would just add an extra issue for anyone trying to debug that.